html-web-page/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <header>
+    <h1>Jelly of the Month Club</h1>
+    <h2>Make all your friends jelly of your stock of jelly!</h2>
+  </header>
+  <p>
+    Here at JOTMC Inc., we are constantly striving to supply our customers with the highest-quality and most unique jellies on a monthly basis. For the jelly lover in your life (or yourself), this is the gift that keeps on giving all year round. We offer three tiers of monthly jelly joy, and each one offers their own perks. Here is a breakdown of each, as well as your monthly price:
+  </p>
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>Basic Plan</th>
+        <th>Deluxe Plan</th>
+        <th>Jelly Giant Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Three new curated jellies per month</td>
+        <td>&check;</td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly jellies come with commemorative mason jars</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free bonus jar of preserves bi-monthly</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free yearly tickets to Jelly Con</td>
+        <td></td>
+        <td></td>
+        <td>&check;</td>
+      </tr>
+    </tbody>
+    <tfoot>
+      <tr>
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tfoot>
+  </table>
+  <p>
+    Like what you see? Become a part of the JOTMC family today!
+  </p>
+  <form>
+    <label for="name">
+      <input type="text" name="name" value="Your Name">
+    </label>
```

Comment:
If the `input` is inside of a `label`, you don't need a `for` attribute. It's either one or the other. So either an `input` is nested inside a `label`, or you give the `label` a `for` attribute that matches the `id` of the input.

Since the `input` also has no text associated with it, you don't even need a `label` at all. I'm going to mention that in our Q&A. You only need a `label` if the `input` has text.